lib/r10k/module/local.rb

```diff
@@ -0,0 +1,21 @@
+require 'r10k/module'
+
+class R10K::Module::Local < R10K::Module::Base
+
+  R10K::Module.register(self)
+
+  def self.implement?(name, args)
+    args.is_a? Symbol and args == :local
```

Comment:
Has this been tested to make sure that this will be respected and not treated as a Forge module?